TGC/localisation/00_events_1000000000.csv

```diff
@@ -13,8 +13,8 @@ EVTOPTA1904191419;An Atlantic connection would greatly boost our trade!;;;;;;;;;
 EVTOPTB1904191419;I don't think we'll ever get that road.;;;;;;;;;;;;;x
 EVTDESC1904191420;Sell Fort Ross to the Mexican citizen John Sutter?;;;;;;;;;;;;;x
 EVTDESC1904191421;$FROMCOUNTRY$ has agreed to sell the Fort Ross trading post to us! Long live $COUNTRY$!;;;;;;;;;;;;;x
-EVTDESC1904191422;The British have come forth to us with a proposal to settle the border between our state of Yucatan and their colony of Belize. In order to prevent future conflict and the native encroachment on our respective territories, they propose that we set the border at the Rio Hondo river. While this would mean we would be renouncing some territorial claims, this treaty would also help us bring the problem of the rebellious Maya in Yucatan under control.;;;;;;;;;;;;;x
-EVTNAME1904191422;The Belize-Yucatan Border;;;;;;;;;;;;;x
+EVTNAME1904191422;The British demand full sovereignity over Belize!;;;;;;;;;;;;;x
```

Comment:
I think I'll start having ptsd from seeing this event's localization come back from the dead

Reply:
yeaaaa that happened again

Reply:
resolved